scs2-definition/src/main/java/us/ihmc/scs2/definition/robot/urdf/URDFTools.java

```diff
@@ -263,6 +253,9 @@ public static URDFModel loadURDFModel(InputStream inputStream,
          if (!parserProperties.parseSensors)
             urdfModel.getGazebos().removeIf(gazebo -> gazebo.getSensor() != null);
 
+         if (parserProperties.handleImplicitJointDefinitions)
```

Comment:
So there is already a flag to not create a revolute twins: `parserProperties.handleImplicitJointDefinitions`